testhelpers/docker_registry.go

```diff
@@ -182,6 +193,39 @@ func (r *DockerRegistry) EncodedLabeledAuth() string {
 	return base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf(`{"username":"%s","password":"%s"}`, r.username, r.password)))
 }
 
+// SetPrivilegesToImage sets the given image name with the provided ImagePrivileges if ImagePrivileges was enabled.
+// For example SetPrivilegesToImage("my-image", NewImagePrivileges(Readable, Writable)) will save "my-image" as a
+// readable and writable image into the registry.
+func (r *DockerRegistry) setPrivilegesToImage(imageName string, privilege ImagePrivileges) {
+	if r.isImagePrivilegesEnable() {
+		r.imagePrivileges[imageName] = privilege
+	}
+}
+
+// MakeReadOnly set the given image name to be readable when the ImagePrivileges feature was enabled
+func (r *DockerRegistry) MakeReadOnly(imageName string) {
```

Comment:
```suggestion
func (r *DockerRegistry) SetReadOnly(imageName string) {
```

Reply:
If you agree with this suggestion, I would change the other functions as well.

Reply:
This is pure syntactic sugar, but maybe you could make this function return the "repo name" for the image, so that [these lines](https://github.com/buildpacks/lifecycle/pull/685/files#diff-c4e4b4a55ebbd71443c5260020b287f3018cb7bb4c65c79e74554b7f97c45656R168-R169) could become 
```
fixtures.readOnlyRegCacheImage = testRegistry.MakeReadWrite(someCacheName) // or SetReadWrite

Reply:
I like making the method to return the repoName. I added in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/c59c94c0966e2826f3e1ce6b48983be810952fee)